<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NdwSecondPage</class>
 <widget class="QWizardPage" name="NdwSecondPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>268</height>
   </rect>
  </property>
  <property name="title">
   <string>Database informations</string>
  </property>
  <property name="subTitle">
   <string>Database name and authentication</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Database</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="dbLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="dbBrowseButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="userLineEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="pswdCheckBox">
     <property name="text">
      <string>Remember password</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="pswdLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="testButton">
     <property name="text">
      <string>Test connection</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="resultLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;- Clic to test</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dbLineEdit</tabstop>
  <tabstop>dbBrowseButton</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>pswdCheckBox</tabstop>
  <tabstop>pswdLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pswdCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pswdLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pswdCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
